pg_lakehouse/benchmarks/README.md

```diff
@@ -19,10 +19,10 @@ brew install docker postgresql
 
 ## Running the Benchmarks
 
-You can then run the benchmarks via:
+You can then run the benchmarks via the `cargo-paradedb` tool with:
 
 ```bash
-./benchmark.sh -t <version> -w <workload>
+cargo paradedb bench hits run -w partitioned
```

Comment:
Can we just delete the `/benchmarks` folder now, and just have this in the `pg_lakehouse` README? Same thing for `pg_search` (but perhaps in a separate PR)? I think the content here needs to be changed too, the arguments got renamed to short/full no?

Reply:
Maybe I'm misunderstanding `short` / `full` thing. When you and @rebasedming were asking for the "full" dataset at the `https://paradedb-benchmarks.s3.amazonaws.com/hits.parquet` url, were you just looking for the `partitioned` workflow that was in the bash script? That is already implemented here, exactly the same interface as the bash script.

Or was it something else that you're looking for?

Reply:
I'm not sure I understand what you're saying exactly. Basically there is a dataset of 100M rows available as a single Parquet file or as 100 partitioned Parquet files for ClickBench. Because that dataset is too big for our CI machines, I've also sampled 5% of it, so we have also a dataset of 5M rows as a single Parquet file and one of 100 partitioned parquet files that add up to 5M rows.

The Bash script had a `-w` script which allows picking between single parquet file and partitioned. And we should have another flag that allows picking between the full dataet or part of the dataset